tests/behat/features/publishing_workflow.feature

```diff
@@ -0,0 +1,42 @@
+@p1 @civictheme @publishing_workflow
+Feature: Publishing workflow
+
+  @api
+  Scenario: Verify the Publishing workflow for civictheme_page
+    Given I am logged in as a user with the "Content Author" role
+    When I visit "node/add/civictheme_page"
+    And I fill in "Title" with "TEST Page Publishing workflow"
+    And I select "draft" from "edit-moderation-state-0-state"
+    And I press "Save"
+
+    When I visit "/admin/content/moderated"
+    And I should see the text "TEST Page Publishing workflow"
+    And I should see the text "Draft"
```

Comment:
This will find all text “draft” on the page

please use another step definition that works with tables